docs/design/hub-and-spoke/switch-validate/switch-validate.md

```diff
@@ -5,9 +5,861 @@
+  },
+  "expected": {
+    "flags": [
+      "STATS"
+    ],
+    "rate": 200,
+    "burst_size": 230
+  },
+  "flags": [
+    "STATS"
+  ],
+  "rate": 300,
+  "burst_size": 200,
+  "cookie": 10,
+  "meter_id": 1
+}
+```
+
+### Validate rules and meters installed on the switch
+
+`GET https://{host}/v1/switches/{switch_id}/validate`
+
+Response payload
+
+```json
+{
+  "groups": {
+    "excess": [
+      <groupInfo>
+    ],
+    "misconfigured": [
+      <groupInfo>
+    ],
+    "missing": [
+      <groupInfo>
+    ],
+    "proper": [
+      <groupInfo>
+    ]
+  },
+  "logical_ports": {
+    "error": "some_error",
+    "excess": [
+      <logicalPortInfo>
+    ],
+    "misconfigured": [
+      <logicalPortInfo>
+    ],
+    "missing": [
+      <logicalPortInfo>
+    ],
+    "proper": [
+      <logicalPortInfo>
+    ]
+  },
+  "meters": {
+    "excess": [
+      <meterInfo>
+    ],
+    "misconfigured": [
+      <meterInfo>
+    ],
+    "missing": [
+      <meterInfo>
+    ],
+    "proper": [
+      <meterInfo>
+    ]
+  },
+  "rules": {
+    "excess": [
+      -9223372036854776000
+    ],
+    "excess-hex": [
+      "800000000000001b"
+    ],
+    "misconfigured": [
+      -9223372036854776000
+    ],
+    "misconfigured-hex": [
+      "800000000000001b"
+    ],
+    "missing": [
+      -9223372036854776000
+    ],
+    "missing-hex": [
+      "800000000000001b"
+    ],
+    "proper": [
+      -9223372036854776000
+    ],
+    "proper-hex": [
+      "800000000000001b"
+    ]
+  }
+}
+```
+
+## v2 API : GET
+
+#### Extended group info (extendedGroupInfo)
+
+```json
+{
+  "flow_id": "10",
+  "flow_path": "e0b0716e-cc59-4ef6-8572-518bdcdc5c72",
+  "excess_group_buckets": [
+    {
+      "port": 2,
+      "vlan": 2,
+      "vni": 2
+    }
+  ],
+  "group_buckets": [
+    {
+      "port": 3,
+      "vlan": 3,
+      "vni": 3
+    }
+  ],
+  "group_id": 10,
+  "missing_group_buckets": [
+    {
+      "port": 4,
+      "vlan": 4,
+      "vni": 4
+    }
+  ]
+}
+```
+
+#### Extended logical port info (extendedLogicalPortInfo)
+
+```json
+{
+  "actual": {
+    "physical_ports": [
+      10,
+      11
+    ],
+    "type": "lag"
+  },
+  "expected": {
+    "physical_ports": [
+      10,
+      11,
+      13
+    ],
+    "type": "bfd"
+  },
+  "physical_ports": [
+    1
+  ],
+  "logical_port_number": 100,
+  "type": "lag"
+}
+```
+
+#### Extended meter info (extendedMeterInfo)
+
+```json
+{
+  "actual": {
+    "flags": [
+      "KBPS"
+    ],
+    "rate": 250,
+    "burst_size": 220
+  },
+  "expected": {
+    "flags": [
+      "STATS"
+    ],
+    "rate": 200,
+    "burst_size": 230
+  },
+  "flags": [
+    "STATS"
+  ],
+  "rate": 300,
+  "burst_size": 200,
+  "cookie": 10,
+  "flow_id": "10",
+  "flow_path": "e0b0716e-cc59-4ef6-8572-518bdcdc5c72",
+  "y_flow_id": "111",
+  "meter_id": 1
+}
+```
+
+#### Extended rule info (extendedRuleInfo)
+
+```json
+{
+  "cookie": -9223372036854776000,
+  "cookie_hex": "800000000000001b",
+  "table_id": 10,
```

Comment:
I offer to add human-readable cookie info. bc, that will be very helpful in fixing prod in case of errors. [link](https://github.com/telstra/open-kilda/blob/develop/docs/design/multi-table-pipelines/cookies.md)